### PR TITLE
HAAR-557: Fixing resources blocked by CSP

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: static/js/autocomplete.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var autocompleteElements = document.getElementsByName('autocompleteElements')[0].content.split(',')
```
Comment (brightonsbox) [inline static/js/autocomplete.js:1]: I've tried to simplify all of these autocomplete scripts and reduce duplication by passing the element in as an argument by using meta tags

File: views/addGroup.njk [hunk 2]
```diff
@@ -61,14 +61,8 @@
...
 {% endblock %}
 
-{% block bodyEnd %}
-  {{ super() }}
+{% block pageScripts %}
+  <meta name="autocompleteElements" content="group"/>
   <script src="/assets/accessible-autocomplete.min.js"></script>
-  <script>
-    accessibleAutocomplete.enhanceSelectElement({
-      selectElement: document.querySelector('#group'),
```
Comment (brightonsbox) [inline views/addGroup.njk:69]: The `group` element is passed in with the meta tag

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**